src/marshmallow/validate.py

```diff
@@ -594,3 +594,47 @@ def __call__(self, value: typing.Sequence[_T]) -> typing.Sequence[_T]:
             if val in self.iterable:
                 raise ValidationError(self._format_error(value))
         return value
+
+
+class NoDuplicates(Validator):
```

Comment:
A more common name for this type of validator is "unique".

Reply:
Thanks for the proposal I'll make the change!